### PR TITLE
✨  Add `UnnecessaryArgumentCamelize`

### PR DESCRIPTION
**Pull request by danielvdao**
### Context

Similar to https://github.com/DmitryTsepelev/rubocop-graphql/pull/102 we've noticed that some people accidentally add `camelize` to `argument` as well. So this PR adds functionality to raise in case someone adds `camelize` to a non-underscore field.

cc @DmitryTsepelev 💚

### CODE REVIEW

File: lib/rubocop/graphql/node_pattern.rb [hunk 1]
```diff
@@ -18,11 +18,15 @@ module NodePattern
         )
       PATTERN
 
-      # @!method argument?(node)
-      def_node_matcher :argument?, <<~PATTERN
+      # @!method argument_definition?(node)
```
Comment (danielvdao) [inline lib/rubocop/graphql/node_pattern.rb:21]: I'll roll this back 🤦🏽‍♂️

**Review by DmitryTsepelev — APPROVED** [review]
LGTM 🙂

File: lib/rubocop/cop/graphql/unnecessary_argument_camelize.rb [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      #
+      #   class UserType < BaseType
+      #     argument :filter, String, required: false, camelize: false
+      #   end
+      #
+      #  # bad
+      #
+      #   class UserType < BaseType
+      #     argument :filter, String, required: false, camelize: false
+      #   end
```
Comment (deecewan) [inline lib/rubocop/cop/graphql/unnecessary_argument_camelize.rb:27]: this "good" and "bad" example are the same. am i missing something?

Comment (DmitryTsepelev) [reply lib/rubocop/cop/graphql/unnecessary_argument_camelize.rb:27]: good point, should be `argument :filter, String, required: false` in the good section, will update master

**Pull request merged**